### DISCUSSION TITLE
Can you stop pushing breaking "updates"?

### DISCUSSION THREAD

**Discussion by DrJLT** [phalcon/cphalcon#15298] [General]
Yesterday, I ran a routine update on my servers, and linux updated **php-phalcon4:arm64** from 4.0.6 to 4.1.0.

I didn't notice the problem until half an hour later: my site was giving out 500 errors because the new php-phalcon4:arm64 4.1.0 doesn't load.

It took me almost an hour to fix it, by removing the php-phalcon4 and installing php7.4-phalcon4.

With this, I also noticed that templates stop generating themselves when 'always' and 'stat' in Volt parameters are set to false. This shouldn't be an issue, but somehow this new mofo package deleted my rendered view files. Leaving my website to return 200 code and a blank page across the site.

If you can't make it better, please just stop. It was working fine.

I'll rewrite everything in Golang this year. Can't stand this anymore.

**Comment by Jeckerson** [comment]
Hello.

This is existing process of rolling updates via OS package manager. It is not perfect, but nothing changed from this perspective for a long time. There are some plans to try to fix it little bit due similar cases like that, but your problem is more related to you itself...

> from 4.0.6 to 4.1.0.

My suggestion is to check changelog of tool that you want to upgrade before, not after.

> It took me almost an hour to fix it, by removing the php-phalcon4 and installing php7.4-phalcon4.

Also, my suggestion is to configure **local** or **dev** environment to test your stuff before make release, or at least have **CI** with some tests.

> It was working fine.

Then why you updated? You can remove from source list after install specific version and have always the same version.
